### ISSUE TITLE
English Dub Subtitles Setting Suggestion 

### ISSUE THREAD

**Issue by Sickhead01** [Goldenfreddy0703/Otaku#210]
Can a settings option be added to default to enabling Signs Only/Signs & Songs for English dub episodes? A lot of torrents start with full subtitle enabled and i don't want that. And i can't use Kodi's preferred language setting because that now forces me to turn off subtitles for everything i watch in other addons such as Umbrella. And even if do use "Forced only" in the preferred Kodi language settings, I now have to enable the subtitles manually in English sub episodes. I want to be able to watch things without having to manually turn on or off subtitles all the time.

**Comment by Goldenfreddy0703** [comment]
![Screenshot_20240205_004020_Kodi.jpg](https://github.com/Goldenfreddy0703/Otaku/assets/62456796/48b6c905-28c3-4f09-927b-d049f8b81498)

Hey there, if im correct we should have an option that says Enabled Subtitles for Dub Streams if that helps. I'm still confused on what you mean but will look into it tomorrow.

**Comment by Sickhead01** [comment]
I already have that enabled but I wanted it to be able to specify betweem full subtitles or just signs. Even a "Forced only" option would be nice, and have the preferred subtitle language for dub and sub seperate, so i can select "forced only" for dub and "english" for sub.

**Comment by Goldenfreddy0703** [comment]
i mean i understand but we don't have control over that unfortunately, we only control the languages of the subtitles. Its not possible to have it select signs or songs but about the preferred subtitle language for dub and sub separate, its possible to code that but will take some time to figure out again sense its been awhile sense i worked with subtitle code. Will looks into it hopefully

**Comment by Goldenfreddy0703** [comment]
Hey so actually from thinking about how to code separate subs and dubs, we actually have it hard coded where subtitles depend on what audio you have chosen. We don't have a way to knowing if the user has picked sub stream or dub steam which is unfortunate, if we did then we would have added settings like this a long time ago. 

![code](https://github.com/Goldenfreddy0703/Otaku/assets/62456796/97319b58-807b-4dad-8002-34ba1d94c25f)
This is our code for how we coded the audio and subtitles and we have it hard coded where the audio depends on what subtitles is offered and if its disabled or enabled. This code is very complicated and i really don't see how we can add your suggestion.

I'm gonna go ahead and close this but if a contributor can figure out this complicated logic and code a setting like this in then may we can open this back up and work on it. 

Thank you
